workbench-core/swb-common-ui/src/components/BaseLayout.tsx

```diff
@@ -21,14 +21,15 @@ export interface LayoutProps {
   activeHref?: string;
 }
 
-export default function Layout({
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
not blocking, but it might be nice to alter the naming convention rule for this package to allow function names to begin with capitals so we dont need to include this ignore everywhere